CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 
 - Add automatic PII scrubbing to `logentry.params`. ([#2956](https://github.com/getsentry/relay/pull/2956))
 - Avoid producing `null` values in metric data. These values were the result of Infinity or NaN values extracted from event data. The values are now discarded during extraction. ([#2958](https://github.com/getsentry/relay/pull/2958))
-- Process user reports in separate processing group. ([#2981](https://github.com/getsentry/relay/pull/2981))
+- Add user reports to a correct processing group. ([#2981](https://github.com/getsentry/relay/pull/2981), [#2984](https://github.com/getsentry/relay/pull/2984))
```

Comment:
nit: "separate" seems more explicit than "correct", could we find another term to make the changelog a bit more clear?

Reply:
I will think about this a bit hot to rephrase it.